brigade/core/__init__.py

```diff
@@ -241,8 +241,20 @@ def InitBrigade(config_file="", dry_run=False, **kwargs):
     inv_class_name = conf.inventory.split(".")[-1]
     module = importlib.import_module(module_path)
     inv_class = getattr(module, inv_class_name)
+    inv_class_args = getattr(conf, inv_class_name, {})
```

Comment:
ok, a priori looks good. I am going to suggest a few changes now that you are at it to make this functionality a bit cleaner.

1. First, I'd add function to the `config.py` to do most of this magic. Something like:

```
def resolve_import_from_string(import_path):
       if not import_path or callable(import_path):
            return import_path
       module_name = ".".join(import_path.split(".")[:-1])
       obj_name = import_path.split(".")[-1]
       module = importlib.import_module(module_name)
       return getattr(module, obj_name)
```
(approx. this may or may not work :P)

2. Then I'd move the inventory and transform_function resolution to `Config.__init__`. Something like:

```
inventory = resolve_import_from_string(kwargs.get("inventory"))
kwargs["inventory"] = inventory

transform_function = resolve_import_from_string(kwargs.get("transform_function"))
kwargs["transform_function"] = transform_function
``` 

3. Finally, `InitBrigade` should be able to just do:

```
inv_class = conf.inventory
inv_args = getattr(conf, inv_class.__name__, {})
transform_function = conf.transform_function 
inv = inv_class(transform_function=transform_function, **inv_args)
``` 

I think this should make it cleaner and also make this solution more generic so anyone using the `Config` class could benefit of this regardless of if they are using `InitBrigade` or their own code to initialize all this.